Js/app.js

```diff
@@ -0,0 +1,14 @@
+const hamburger = document.querySelector('.menu-icon');
+const closeMenu = document.querySelector('.menu-cross');
+const navItems = document.querySelectorAll('.mobile-menu a');
+
+function toggleMenu() {
```

Comment:
- [ ] When the user clicks (or taps) the hamburger button, the mobile menu appears, this functionality is not working, please fix it so that the mobile menu can be displayed.

Reply:
Hi Ismail, 

Thanks for the great review but I seem not to have understood your comment.

look forward to your communications
Regards
Patrick

Reply:
@Kirabo19, When you click on the hamburger button the menu does not display at all

Reply:
@Ismailco Hi, I am a collaborator for this project and as I can check everything is working as expected and to the point! I'm not sure why you are facing that problem but I will make a latest push to this branch and I request you to check it again!

Reply:
@samWfelice, The same issue persists, you can check the JavaScript code and fix it, I don't really know what's the problem here